libcxx/test/libcxx/thread/thread.lock/thread.lock.scoped/nodiscard.verify.cpp

```diff
@@ -0,0 +1,34 @@
+//===----------------------------------------------------------------------===//
```

Comment:
I think this should just be part of `libcxx/test/libcxx/diagnostics/mutex.nodiscard.verify.cpp`.